src/webgl/material.js

```diff
@@ -301,6 +303,36 @@ p5.prototype.resetShader = function() {
  * </code>
  * </div>
  *
+ * <div>
+ * <code>
+ *
+ * // In the example below, you can see that the canvas contains an ambient light and a point light.
+ * // None of these have any effect on the Normal Material.
+ *
+ * let angle = 0;
+ * function setup() {
+ *    createCanvas(100, 100, WEBGL);
+ * }
+ * function draw() {
+ *    background(51);
+ *    let locX = mouseX - width / 2;
+ *    let locY = mouseY - height / 2;
+ *
+ *    // Lights:
+ *    ambientLight(50);
+ *    pointLight(250, 250, 250, locX, locY, 50);
+ *
+ *    // Object:
+ *    rotateY(angle);
+ *    rotateZ(angle);
+ *    noStroke();
+ *    normalMaterial();
+ *    box(40);
+ *
+ *    angle += 0.01;
+ * }
```

Comment:
these reference examples should be as simple and concise as possible to reduce chance of confusion. 
let's make two examples. the first without any mouse control, just a static light. the second with the mouse control.